test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -207,6 +208,144 @@ public void NormalizerParameters()
+                                new[] {new InputOutputColumnPair("float1mv", "float1"), new InputOutputColumnPair("float4mv", "float4"),
+                                    new InputOutputColumnPair("double1mv", "double1"), new InputOutputColumnPair("double4mv", "double4")}))
+                    .Append(context.Transforms.NormalizeLogMeanVariance(
+                                new[] {new InputOutputColumnPair("float1lmv", "float1"), new InputOutputColumnPair("float4lmv", "float4"),
+                                    new InputOutputColumnPair("double1lmv", "double1"), new InputOutputColumnPair("double4lmv", "double4")}))
+                    .Append(context.Transforms.NormalizeSupervisedBinning(
+                                new[] {new InputOutputColumnPair("float1nsb", "float1"), new InputOutputColumnPair("float4nsb", "float4"),
+                                    new InputOutputColumnPair("double1nsb", "double1"), new InputOutputColumnPair("double4nsb", "double4")}));
+
+            var data = loader.Load(dataPath);
+
+            var transformer = est.Fit(data);
+            var transformers = transformer.ToImmutableArray();
+            var floatAffineData = ((NormalizingTransformer)transformers[0]).Columns[0].ModelParameters as NormalizingTransformer.AffineNormalizerModelParameters<float>;
+            Assert.Equal(0.12658228f, floatAffineData.Scale);
+            Assert.Equal(0, floatAffineData.Offset);
+
+            var floatAffineDataVec = ((NormalizingTransformer)transformers[0]).Columns[1].ModelParameters as NormalizingTransformer.AffineNormalizerModelParameters<ImmutableArray<float>>;
+            Assert.Equal(4, floatAffineDataVec.Scale.Length);
+            Assert.Empty(floatAffineDataVec.Offset);
+
+            var doubleAffineData = ((NormalizingTransformer)transformers[0]).Columns[2].ModelParameters as NormalizingTransformer.AffineNormalizerModelParameters<double>;
+            Assert.Equal(0.12658227848101264, doubleAffineData.Scale);
+            Assert.Equal(0, doubleAffineData.Offset);
+
+            var doubleAffineDataVec = ((NormalizingTransformer)transformers[0]).Columns[3].ModelParameters as NormalizingTransformer.AffineNormalizerModelParameters<ImmutableArray<double>>;
+            Assert.Equal(4, doubleAffineDataVec.Scale.Length);
+            Assert.Empty(doubleAffineDataVec.Offset);
+
+            var floatBinData = ((NormalizingTransformer)transformers[1]).Columns[0].ModelParameters as NormalizingTransformer.BinNormalizerModelParameters<float>;
+            Assert.True(35 == floatBinData.UpperBounds.Length);
+            Assert.True(34 == floatBinData.Density);
+            Assert.True(0 == floatBinData.Offset);
+
+            var floatBinDataVec = ((NormalizingTransformer)transformers[1]).Columns[1].ModelParameters as NormalizingTransformer.BinNormalizerModelParameters<ImmutableArray<float>>;
+            Assert.True(4 == floatBinDataVec.UpperBounds.Length);
```

Comment:
Could you please check at least the first and the last values in `UpperBounds` and other array-like fields?